src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookup.class.php

```diff
@@ -564,17 +564,21 @@ public function toString()
             return '';
         }
         $tableName = $this->GetConnectedTableName();
-        $nameColumn = 'name';
+        $identifyingColumnName = 'name';
+
+        if (false === TCMSTableConf::FieldExists($tableName, $identifyingColumnName)) {
+            $identifyingColumnName = 'id';
+        }
```

Comment:
else: probably the id should / must be present anyway - otherwise entries cannot be really safely identified

Reply:
Do we really need to identify records uniquely here? The method is just for printing a readable representation of the field value.

In the query we select the ID field but don't use it, so I think we should drop it from the query.

Reply:
Check removed.